22.RaytracedAO/CommandLineHandler.hpp

```diff
@@ -16,69 +16,77 @@ constexpr std::string_view helpMessage = R"(
 
 Parameters:
 -SCENE=sceneMitsubaXMLPathOrZipAndXML
--TERMINATE
 
 Description and usage: 
 
 -SCENE:
 	some/path extra/path which will make it skip the file choose dialog
-
--TERMINATE:
-	which will make the app stop when the required amount of samples has been renderered (its in the Mitsuba Scene metadata) and obviously take screenshot when quitting
 	
 Example Usages :
-	raytracedao.exe -SCENE=../../media/kitchen.zip scene.xml -TERMINATE
-	raytracedao.exe -SCENE="../../media/my good kitchen.zip" scene.xml -TERMINATE
-	raytracedao.exe -SCENE="../../media/my good kitchen.zip scene.xml" -TERMINATE
-	raytracedao.exe -SCENE="../../media/extraced folder/scene.xml" -TERMINATE
+	raytracedao.exe -SCENE=../../media/kitchen.zip scene.xml
+	raytracedao.exe -SCENE="../../media/my good kitchen.zip" scene.xml
+	raytracedao.exe -SCENE="../../media/my good kitchen.zip scene.xml"
+	raytracedao.exe -SCENE="../../media/extraced folder/scene.xml"
```

Comment:
keep the help-text and documentation up to date

Reply:
Resolved.